src/mgos_imu_bmm150.c

```diff
@@ -0,0 +1,389 @@
+/*
+ * Copyright 2018 Google Inc.
```

Comment:
ThIs file is not copyright Google, unless the author works for Google. Can you make sure to put the correct holder here?

Reply:
done